unbxdAutosuggest.js

```diff
@@ -1095,13 +1104,19 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 			return arr.indexOf(autosuggest) === -1 ? arr.push(autosuggest) : false;
 		}
 		, getfilteredPopularProducts: function () {
-			var self = this,
-				url_path = this.getHostDomainName() + this.options.APIKey + "/"
+			var self = this;
+			
```

Comment:
This new line is not required

Reply:
You can remove this change since `var` is sufficient for multiple variables declaration.

Reply:
done